app/views/admin/access_overviews/index.haml

```diff
@@ -14,24 +14,25 @@
         %p.access-description
           Each
           %span.access-control.color-tag<
-            = link_to 'access control', '#', tabindex: '0', data: {container: '.access-overview', toggle: :popover, placement: :top, html: true, trigger: :focus, content: "<p>Accss Controls are used to grant users access to data and permissions.  Each access control consists of one Role, one User Group, and one Collection. Access is governed by the intersection of permission from the role, entities from the collection, and users from the user group.</p><p>#{link_to 'Manage Access Controls', admin_access_controls_path}</p>", title: 'Access Control'}
```

Comment:
Typo a few lines above: `goverend` => `governed`

Reply:
Also suggest adding a comma:
`are used to grant users access to data*,* and `

Reply:
That's what I get for copying out of figma I guess.